server/modules/media/controllers/index.js

```diff
@@ -2,13 +2,14 @@ import { Router } from 'express';
 
 import getMediaById from './get-media-by-id';
 import getSignedURL from './get-signed-url';
+import getMediaUrl from './get-media-url';
 
 import { authenticate } from '../../../api/middlewares';
 
 const router = Router();
 
+router.get('/', authenticate(), getSignedURL);
+router.get('/media-url', authenticate(), getMediaUrl);
```

Comment:
```suggestion
router.get('/url', authenticate(), getMediaUrl);
```

because the rout is `/media/...` so its already on media endpint